scipy/special/stirling2.h

```diff
@@ -0,0 +1,122 @@
+#ifndef STIRLING_H
+#define STIRLING_H
+
+#include <complex.h>
+#include <math.h>
+#include <stdio.h>
+#include <stdint.h>
+#include <stdlib.h>
+
+#include "special/lambertw.h"
+
+
+/*     Stirling numbers of the second kind
+ *
+ * SYNOPSIS: Stirling numbers of the second kind count the
+ *  number of ways to make a partition of n distinct elements
+ *  into k non-empty subsets.
+ *
+ * DESCRIPTION: n is the number of distinct elements of the set
+ *  to be partitioned and k is the number of non-empty subsets.
+ *  The values for n < 0 or k < 0 are interpreted as 0. If you
+ *
+ * ACCURACY: The method returns a double type.
```

Comment:
I think the  `ACCURACY` field is describing the expected relative error in different parameter regimes. Like this from `cephes/beta.c`.

```C
/* ...
 * ACCURACY:
 *
 *                      Relative error:
 * arithmetic   domain     # trials      peak         rms
 *    IEEE       0,30       30000       8.1e-14     1.1e-14
 *  ... */
```

Reply:
I'll delete the ACCURACY section of the comment